Day-6/src/components/GithubCard.js

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react";
+
+const GithubCard = ({ data }) => {
+  const [teamData, setTeamData] = useState([]);
+  console.log(data);
+
+  useEffect(() => {
+    displayUserData();
+  }, []);
+
+  const fetchUserData = async (url) => {
```

Comment:
Nice. This can be used in other places too. Can add this to the utils file.

Reply:
yes done!